lib/zold/version_file.rb

```diff
@@ -0,0 +1,32 @@
+require 'semantic'
+
+module Zold
+  # Read and write .zoldata/version.
+  class VersionFile
+    def initialize(path)
+      @path = path
+    end
+
+    def save(version)
+      File.open(@path, 'w') do |file|
+        file.puts(version)
+      end
+    end
+
+    def apply(version)
+      save(version) if yield
```

Comment:
@botanicus I don't think this implementation is correct at the moment. It will apply _all_ versions on each run. This is not what we need. We have to check what is the current version, compare it with `version` and only then apply or not. I would suggest to replace this entire class (`VersionFile`) with a stub, which will do nothing in `apply` except from logging `"Version #{version} doesn't need to be applied"`. And puzzle it. This implementation is wrong.